discovery-infra/test_infra/controllers/node_controllers/host.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from test_infra.controllers.node_controllers import ssh
+
+
+class Host:
+
+    def __init__(self, name, node_controller, private_ssh_key_path=None, username="core"):
+        self.name = name
+        self.private_ssh_key_path = private_ssh_key_path
+        self.username = username
+        self.node_controller = node_controller
+        self._ips = []
+        self._macs = []
+
+    @property
+    def is_active(self):
+        return self.node_controller.is_active(self.name)
+
+    def _set_ips_and_macs(self):
+        self._ips, self._macs = self.node_controller.get_node_ips_and_macs(self.name)
+
+    @property
+    def ips(self):
+        if not self._ips:
+            self._set_ips_and_macs()
```

Comment:
maybe retrieve the IPs every time and not cache it, in case the DHCP lease was not renewed between restarts?

Reply:
hmm, possible. I didn't see them changed

Reply:
added comment. We will need to handle it one day

Reply:
will be much easier if we will move to hosts class

Reply:
sure